README.md

```diff
@@ -95,6 +92,16 @@ Inside **`View.elm`**, we define the view for our model and set up any event han
 
 **`Flags.elm`** contains a decoder for the flags of the app. We aim to keep our decoders basic and so decode into a special `Flags` type that mirrors the structure of the raw JSON instead of the structure of the `Model` type. The `Flags` and `Model` modules should not depend on each other.
 
+**`Main.elm`** is our entry file. Here, we import everything from the other files and actually connect everything together.
+
+It calls `Html.programWithFlags` with:
+- `init`, defined in `Main`, runs `Flags.decodeFlags` and turns the resulting `Flags` type into a `Model`.
+- `update` is `Update.update >> Util.batchUpdate`. That last part collapses the `List (Cmd Msg)` into a single `Cmd Msg`.
+- `view` is simply `View.view`.
+- `subscriptions`, defined in `Main`, contains any subscriptions this app relies on.
+
+Additionally we setup ports for interop with JS in this file. We run elm-make on this file to generate a JS file that we can include elsewhere.
```

Comment:
I guess that's necessary because `Update` can depend on a port and it will not be able to import them from `Main.elm`. Do we want to write down the use of a separate `Ports.elm` as the default, or something we do on an as-needed basis?